platform/mds/mds-web/src/main/resources/webapp/partials/widgets/entityInstances.html

```diff
@@ -34,7 +34,7 @@ <h4 class="frame-title">{{selectedEntity.module}} <span ui-if="selectedEntity.mo
             <button ng-show="showFilters && showFiltersButton" type="button" class="btn btn-default" id="mds-filters">
                 <i class="fa fa-filter"></i>&nbsp;{{msg('mds.filters')}}
             </button>
-            <button id="lookupDialogButton" ng-click="showLookupDialog()" type="button" class="btn btn-default" ng-show="showLookupButton">
+            <button id="lookupDialogButton" ng-click="showLookupDialog()" type="button" class="btn btn-default" ng-show="showLookupButton" ng-disabled="entityAdvanced.indexes == 0">
```

Comment:
According to the ticket description, the button should be invisible instead of being grayed out. Please use ng-if instead.